yaml.go

```diff
@@ -33,8 +33,19 @@ type JSONOpt func(*json.Decoder) *json.Decoder
 // Unmarshal converts YAML to JSON then uses JSON to unmarshal into an object,
 // optionally configuring the behavior of the JSON unmarshal.
 func Unmarshal(y []byte, o interface{}, opts ...JSONOpt) error {
+	return unmarshal(yaml.Unmarshal, y, o, opts)
+}
+
+// UnmarshalStrict is like Unmarshal except that any mapping keys that are
+// duplicates, will result in an error.
```

Comment:
remove the comma here

Reply:
Off topic: For this feature branch and pull request, do you prefer me fixing the issue by making several commits on top of this one or do you prefer that I fix this single commit and replace it with a force-push?

Reply:
We can squash-and-merge when it's approved, so feel free to stack additional commits for now. Or use a single commit with force push. Your call.

Reply:
Thanks! I'll add commits on top to honor this conversation.